### ISSUE TITLE
change json results

### ISSUE THREAD

**Issue by eroux** [BuddhistDigitalResourceCenter/lds-pdi#28]
The current json output looks like:

```json
  "headers" : [ "s", "lit" ],
  "rows" : [ {
    "dataRow" : {
      "s" : "<a href=/resource/P3393> P3393</a>",
      "lit" : "'od zer/@bo-x-ewts"
    }
  }, {
```

but what an API consumer wants is the URI of the resource, not an html link. So we have several solutions.

### Solution 1

A small change:

```json
  "headers" : [ "s", "lit" ],
  "rows" : [ {
    "dataRow" : {
      "s" : "<http://purl.bdrc.io/resource/P3393>",
      "lit" : "'od zer/@bo-x-ewts"
    }
  ...
```

(not sure about the `<...>`)

### Solution 2

add prefixes, as in:

```json
  "headers" : [ "s", "lit" ],
  "prefixes": {"bdr": "http://purl.bdrc.io/resource/"}
  "rows" : [ {
    "dataRow" : {
      "s" : "bdr:P3393",
      "lit" : "'od zer/@bo-x-ewts"
    }
  ...
```

### Solution 3

just transmit the SPARQL result, which looks just fine:

```json
  "headers" : [ "s", "lit" ],
  "prefixes": {"bdr": "http://purl.bdrc.io/resource/"}
  "results": {
    "bindings": [
      {
        "s": { "type": "uri" , "value": "http://purl.bdrc.io/resource/P3393" } ,
        "lit": {  "type": "literal" , "xml:lang": "bo-x-ewts" , "value": "'od zer/" } ,
      } ,
   ...
```

I think Solution 3 is the most solid one.

**Comment by MarcAgate** [comment]
Implemented in commit df35a6a

**Comment by eroux** [comment]
Thanks, not sure we really need to add prefixes, but the datatypes are missing from the literal fields, they appear in some SPARQL results like the following:

```json
{
  "head": {
    "vars": [ "p" ]
  } ,
  "results": {
    "bindings": [
      {
        "p": { "type": "literal" , "datatype": "http://www.w3.org/2001/XMLSchema#integer" , "value": "1" }
      } ,
```

**Comment by MarcAgate** [comment]
Ok. added Literal datatypes URI in json Results

**Comment by eroux** [comment]
Looking at the code I think it will output 

```json
      {
        "p": { "type": "http://www.w3.org/2001/XMLSchema#integer" , "value": "1" }
      } ,
```
 
instead of:

```json
      {
        "p": { "type": "literal" , "datatype": "http://www.w3.org/2001/XMLSchema#integer" , "value": "1" }
      } ,
```

? If so, I think it shouldn't, I see some virtue in using the exact same format as the SPARQL query...

**Comment by MarcAgate** [comment]
Ok ! Implemented in commit f41a720
